src/main/java/at/hannibal2/skyhanni/config/features/gui/GUIConfig.java

```diff
@@ -140,6 +140,16 @@ public class GUIConfig {
     @ConfigLink(owner = GUIConfig.class, field = "tpsDisplay")
     public Position tpsDisplayPosition = new Position(10, 10, false, true);
 
+    @Expose
+    @ConfigOption(name = "Ping Display", desc = "Show your current ping, like in Skytils.")
```

Comment:
I dont think this needs to say like skytils, as skytils does not show your ping on the screen